.Lib9c.Tests/xunit.runner.json

```diff
@@ -0,0 +1,4 @@
+{
+        "$schema": "https://xunit.net/schema/current/xunit.runner.schema.json",
+        "maxParallelThreads": 32
+}
```

Comment:
Is this change intentional?

Reply:
Nope, it seems a mistake.

Reply:
Amended. https://github.com/greymistcube/lib9c/pull/1/commits/b86894bb798dc07a9668d5e04a5d26b7ec288576